status-im/status-react
# [ISSUE #2883] Migrate wallet/request to latest mockups

fixes #2883
fixes #3042
fixes #3220
improves #3196

### Summary:

Migrate wallet/request to latest mockups while relying on components.

### Steps to test:
- Open Status
- Navigate wallet / 
- Screen should work as previously and respect mockups

status: ready
